client/src/javascript/stores/ConfigStore.js

```diff
@@ -26,7 +9,7 @@ class ConfigStoreClass extends BaseStore {
   }
 
   getBaseURI() {
-    return this.userConfig.baseURI || '/';
+    return '';
```

Comment:
Minimal change was the easiest

Reply:
Yeah, this PR should be updated to remove all references to `baseURI`.